cloudvolume/datasource/precomputed/metadata.py

```diff
@@ -658,4 +662,51 @@ def add_scale(self, factor, encoding=None, chunk_size=None, info=None):
     if not preexisting:    
       info['scales'].append(newscale)
 
-    return newscale
\ No newline at end of file
+    return newscale
+
+  def lock_mips(self, mips):
+    """
+    Establishes a write lock on the specified mip levels.
+    The lock is written to the cloud info file.
+    """
+    mips = lib.toiter(mips)
+    if max(mips) > max(self.available_mips):
+      raise ValueError("Cannot lock a mip level that doesn't exist. Highest mip: {} Got: {}".format(
```

Comment:
Maybe I want to prevent the generation of additional downsamples? 😈

Reply:
Is that realistic? I'm not sure why you would care as you could delete them.

Reply:
Probably not, no.